Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi8/python-38 AS demo
+
+COPY ./ /trestle-bot
+
+WORKDIR /trestle-bot
+
+# Install dependencies
+RUN  python3.8 -m pip install --upgrade pip \
+     && pip install poetry \ 
```

Comment:
We are going to want to pin to a specific version of poetry. Perhaps set a default ARG and allow the version to be overridden during the build.